docs/design.md

```diff
@@ -30,6 +30,15 @@ parameters:
     type: int
     default: 3306
 ```
+Or with an empty parameters field
+```yaml
+name: fusor/etherpad-apb
+description: Note taking web application
+bindable: true
+async: optional
+parameters: []
```

Comment:
I'm not sure if there's one that works with no params.  But, we definitely want to put an example here that does because folks will look at it and try to use it.